app/(landing)/views/social-proof/social-proof.module.scss

```diff
@@ -0,0 +1,117 @@
+$main-seciton-width: 70%;
+$secondary-seciton-width: calc(100% - $main-seciton-width);
+
+.container {
+    background-color: #fff;
+    padding: var(--padding-900) var(--width-gap);
+    text-align: left;
+    color: #333;
+    width: 100%;
+}
+
+.socialProof {
+    max-width: var(--width-900);
+    margin: 0 auto;
+    align-items: normal;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    align-content: flex-start;
+    justify-content: space-between;
+    min-height: auto;
+
+    h3 {
+        font-size: var(--font-size-600);
+        font-weight: var(--font-normal);
+        line-height: var(--line-height-600);
+        color: var(--color-text-500);
+        margin-bottom: var(--margin-100);
+        text-align: left;
+    }
+
+    h2 {
+        font-size: var(--font-size-900);
+        line-height: var(--line-height-750);
+        font-weight: var(--font-medium);
+        margin-bottom: var(--margin-300);
+        color: var(--color-text-800);
+        text-align: left;
+        max-width: $main-seciton-width;
+        @media (max-width: 1000px) {
+            max-width: 100%;
+        }
+    }
```

Comment:
I think existing h2 have proper styles, no need to override them

Reply:
Applied default styles with gradient